torch/_torch_docs.py

```diff
@@ -3137,12 +3137,28 @@ def merge_dicts(*dicts):
 The dividend and divisor may contain both for integer and floating point
 numbers. The remainder has the same sign as the dividend :attr:`input`.
 
-When :attr:`other` is a tensor, the shapes of :attr:`input` and
-:attr:`other` must be :ref:`broadcastable <broadcasting-semantics>`.
+Supports :ref:`broadcasting to a common shape <broadcasting-semantics>`,
+:ref:`type promotion <type-promotion-doc>`, and integer and float inputs.
+
+.. note::
+
+    When the divisor is zero, returns ``NaN`` for common dtype as
+    floating-point on both CPU and GPU; raises ``RuntimeError`` for
+    common dtype as integral on CPU;
```

Comment:
"for common dtype as integral on CPU;" -> "for integer division by zero on CPU."